modules/gitops-installing-argocd-cli-on-linux-using-rpm.adoc

```diff
@@ -0,0 +1,137 @@
+
+.Prerequisites
+
+* You have an active {OCP} subscription on your Red Hat account.
+* You have root or `sudo` privileges on your local system.
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
++
+[source,terminal]
+----
+# subscription-manager register
+----
+
+. Pull the latest subscription data:
++
+[source,terminal]
+----
+# subscription-manager refresh
+----
+
+. List the available subscriptions:
++
+[source,terminal]
+----
+# subscription-manager list --available --matches '*gitops*'
+----
+
+. In the output for the previous command, find the pool ID for your {OCP} subscription and attach the subscription to the registered system:
++
+[source,terminal]
+----
+# subscription-manager attach --pool=<pool_id>
+----
+
+. Enable the repositories required by {gitops-title}:
++
+* For RHEL 8
+** Linux (x86_64, amd64)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-8-x86_64-rpms"
+----
++
+** Linux on {ibmzProductName} and {linuxoneProductName} (s390x)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-8-s390x-rpms"
+----
++
+** Linux on {ibmpowerProductName} (ppc64le)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-8-ppc64le-rpms"
+----
++
+** Linux on ARM (aarch64, arm64)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-8-aarch64-rpms"
+----
++
+* For RHEL 9
+** Linux (x86_64, amd64)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-9-x86_64-rpms"
+----
++
+** Linux on {ibmzProductName} and {linuxoneProductName} (s390x)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-9-s390x-rpms"
+----
++
+** Linux on {ibmpowerProductName} (ppc64le)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-9-ppc64le-rpms"
+----
++
+** Linux on ARM (aarch64, arm64)
++
+[source,terminal]
+----
+# subscription-manager repos --enable="gitops-1.12-for-rhel-9-aarch64-rpms"
+----
+
+. Install the `openshift-gitops-argocd-cli` package:
++
+[source,terminal]
+----
+# yum install openshift-gitops-argocd-cli
+----
+
+. After you install the {gitops-shortname} `argocd` CLI, verify that it is available by running the following command:
++
+[source,terminal]
+----
+$ argocd version --client
+----
++
+.Example output
+[source,terminal]
+----
+argocd: v2.9.5+f943664
+  BuildDate: 2024-02-15T05:19:27Z
+  GitCommit: f9436641a616d277ab1f98694e5ce4c986d4ea05
+  GitTreeState: clean
+  GoVersion: go1.20.10
+  Compiler: gc
+  Platform: linux/amd64
+  ExtraBuildInfo: openshift-gitops-version: 1.12.0, release: 0015022024 <1>
+----
+<1> The build information of {gitops-title} built by Red Hat.
++
+[NOTE]
+====
+The example output is just for reference. The actual details might be different based on the version of the {gitops-shortname} `argocd` CLI client installed.
```

Comment:
Do we need to state this if the output is explicitly labelled as "Example output"?